python/singa/sonnx.py

```diff
@@ -258,8 +258,8 @@ def to_onnx_model(inputs, y, model_name='sonnx'):
     dependency = autograd.infer_dependency(y.creator)
     ready = deque([y.creator])
 
-    def output_name(op):
-        return '{}'.format(op.name)
+    def output_name(op,extra=''):
+        return '{}'.format(op.name+str(extra))
```

Comment:
extra is a str? then why do you the type conversion?

Reply:
default is string, but if pass int to the function, it can convert to string automatically